freud/diffraction.pyx

```diff
@@ -46,6 +46,15 @@ cdef class DiffractionPattern(_Compute):
     as a multiplication in Fourier space. The computed diffraction pattern
     can be accessed as a square array of shape ``(output_size, output_size)``.
 
+    The :math:`\vec{k}=0` peak is always located at index
+    ``(output_size // 2, output_size // 2)`` and is normalized to have a value
+    of :math:`S(\vec{k}=0) = 1` (not :math:`N`, a common convention). The
+    remaining :math:`\vec{k}`-vectors are computed such that each peak in the
+    diffraction pattern satisfies the relationship :math:`\vec{k} \cdot
+    \vec{R} = 2\piN` for some integer :math: `N` and lattice vector of
+    the system :math: `R` (see
+    `this article <https://en.wikipedia.org/wiki/Reciprocal_lattice>`).
```

Comment:
Needs some syntax fixes. You can see the live docs rendered here: https://freud--777.org.readthedocs.build/en/777/modules/diffraction.html#freud.diffraction.DiffractionPattern
```suggestion
    \vec{R} = 2 \pi N` for some integer :math:`N` and lattice vector of
    the system :math:`R`. See the
    `reciprocal lattice Wikipedia page <https://en.wikipedia.org/wiki/Reciprocal_lattice>`__
    for more information.
```
You may also need to add `# noqa: E501` to the end of this docstring since that line with the link is fairly long. Search "E501" to see examples in the rest of the code.